lerna.json

```diff
@@ -10,5 +10,5 @@
   "packages": [
     "packages/*"
   ],
-  "version": "1.0.0"
+  "version": "2.0.0"
```

Comment:
I have more breaking changes…

So we don't care if this keeps going up super quick?